### PR TITLE
Text on hover fixed in Blockrain activity

### LINKED ISSUE #627: Wrong text appears upon hovering on buttons in Blockrain activity
Text should appear as "Rotate Block" instead of "Up Arrow"

blockrain_issue_1

### PR DESCRIPTION
**Pull request by abhishektanwar**
Fixes #627 
blockrain_fix_1

### CODE REVIEW

File: activities/Blockrain.activity/css/activity.css [hunk 1]
```diff
@@ -16,6 +16,138 @@
   background-image: url(../icons/view-fullscreen.svg);
 }
 
+#main-toolbar #help-button {
+  background-image: url(../icons/help.svg);
+}
+
+.tutorial-title {
+	background-color: #808080 !important;
```
Comment (llaske) [inline activities/Blockrain.activity/css/activity.css]: Avoid mixing indentation style in the same file

File: activities/Blockrain.activity/index.html [hunk 2]
```diff
@@ -25,13 +28,14 @@
 
 <body>
 <div id="main-toolbar" class="toolbar">
-  <button class="toolbutton" id="activity-button" title="My Activity"></button>
+  	<button class="toolbutton" id="activity-button" title="My Activity"></button>
```
Comment (llaske) [inline activities/Blockrain.activity/index.html]: Avoid mixing indentation style in the same file

File: activities/Blockrain.activity/lib/tutorial.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                        </div>\
+                        <div class='icon-tutorial-text'>End</div>\
+                    </div>\
+                </div>\
+            </div>",
+            storage: false,
+            backdrop: true,
+            steps: steps
+        });
+		tour.init();
```
Comment (llaske) [inline activities/Blockrain.activity/lib/tutorial.js]: Avoid mixing indentation style in the same file

File: activities/Blockrain.activity/lib/tutorial.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
+define([], function () {
+	var tutorial = {};
+
+	tutorial.start = function() {
+		var steps = [
+			{
+				element: "",
+				orphan: true,
+				placement: "bottom",
+				title: "Blockrain",
```
Comment (llaske) [inline activities/Blockrain.activity/lib/tutorial.js]: All title and content should be localized as explained [here](https://github.com/llaske/sugarizer/blob/dev/docs/tutorial_step9.md#localize-the-tutorial).

File: activities/Blockrain.activity/lib/tutorial.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+				content: "Welcome into the Blockrain activity. Blockrain is a tile-matching puzzle video game."
+            },
+            {
+				element: "#canvas",
+				placement: "right",
+				title: "Rules",
+				content: "You can only move the pieces in specific ways, your game is over if your pieces reach the top of the screen, and you can only remove pieces from the screen by filling all the blank space in a line (horizontally)."
+            },
+			{
```
Comment (llaske) [inline activities/Blockrain.activity/lib/tutorial.js]: Could you put the theme popup at the end of the tutorial?

File: activities/Blockrain.activity/css/activity.css (hunk 1 above)
Comment (abhishektanwar) [reply activities/Blockrain.activity/css/activity.css]: Fixed

**Comment by llaske** [comment]
There is still indentation mix in tutorial.js. Check in your editor:

image

Regarding the theme popup, the idea was not to add a new popup at end to describe the theme but just to put the theme popup at the end of the array.

**Comment by abhishektanwar** [comment]
@llaske I have fixed the indentation issue and moved the theme popup at the end of tutorial. Please Review. Fixes #593

**Comment by llaske** [comment]
Don't answer too quick. There is still issue on the indentation in the tutorial.js file.

**Comment by abhishektanwar** [comment]
@llaske I have made the indentation in tutorial.js similar to the one in #603 .I hope it is fine now

**Comment by llaske** [comment]
Sorry, there is still indentation issue.
If your current IDE is not able to show you difference between TAB and SPACE, I suggest you to change it.

**Comment by abhishektanwar** [comment]
@llaske  My whitespaces were disabled that is why I couldn't figure out what you were pointing to. I have fixed the spaces now . Please check 
image

image

File: activities/Blockrain.activity/lib/sugar-web/activity/activity.js [hunk 6]
```diff
@@ -5,8 +5,10 @@ define(["webL10n",
         "sugar-web/datastore",
 		"sugar-web/presence",
         "sugar-web/graphics/icon",
-        "sugar-web/graphics/activitypalette"], function (
-    l10n, shortcut, bus, env, datastore, presence, icon, activitypalette) {
+        "sugar-web/graphics/activitypalette",
```
Comment (llaske) [inline activities/Blockrain.activity/lib/sugar-web/activity/activity.js]: You're not supposed to change this file, it's a part of the sugar-web library.

Comment (abhishektanwar) [reply activities/Blockrain.activity/lib/sugar-web/activity/activity.js]: Fixed icon at end button and 

> You're not supposed to change this file, it's a part of the sugar-web library.

Fixed

**Comment by abhishektanwar** [comment]
@llaske fixed both the requested changes. Please review

**Comment by abhishektanwar** [comment]
@llaske can you please review the PR?

**Comment by llaske** [comment]
Good job. Thanks.

**Pull request merged**